core/pom.xml

```diff
@@ -150,6 +150,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.python</groupId>
+            <artifactId>jython</artifactId>
+            <version>2.7.0</version>
```

Comment:
Is it possible/feasible to install additional python modules for use with jython? Does the cPython ecosystem place nicely?